src/server/pachyderm_test.go

```diff
@@ -216,6 +216,38 @@ func TestSimplePipeline(t *testing.T) {
 			require.Equal(t, "foo", buf.String())
 		}
 	}
+
+	pipeline = strings.Repeat("x", 59-len(projectName)-1)
```

Comment:
This is fine to test here, but the most ideal situation would be a unit test for validateCreatePipelineRequest or if you want to be more careful, send a request to the API server setup with mockEnv.  There isn't a huge need to boot up pachyderm to have it reject the name.